### DISCUSSION TITLE
Photoprism in GSoC 2022?

### DISCUSSION THREAD

**Discussion by nautatva** [photoprism/photoprism#2142] [Ideas]
I love photoprism and how the team is doing. Now that [Google Summer Of Code Opens Up To Non-Students](https://opensource.googleblog.com/2021/11/expanding-google-summer-of-code-in-2022.html), photoprism could serve as an excellent platform for learning. Wondering if anyone in photoprism has project ideas, and the capacity to mentor students? (Past the ideas phase, it's generally just being available to review work promptly so that the student can keep momentum — so it's not super onerous, but it is a commitment.)

Let me know if you think, Oooh, maybe — happy to chat and work out if it's right, I myself want to start working on photoprism. 😄

**Comment by graciousgrey** [comment]
Thanks a lot for your feedback! 

This sounds very interesting, unfortunately if we lack one thing it's resources.

Maybe next year :)
